app/views/movies/_movie_add_to_another_list.html.erb

```diff
@@ -1,10 +1,17 @@
 
   <% if current_user.lists_except_movie(movie).count > 0 %>
     <%= form_for(Listing.new, class: "form-class", id: "modal-add-to-other-list", remote: true) do |f| %>
-      <%= f.collection_select :list_id, current_user.lists_except_movie(movie), :id, :name, { prompt: "Add to Another List" }, {class: "form-control"} %>
+      <%= text_field_tag  'list_dropdown',
```

Comment:
this and the other form are the instances of duplication mentioned in the PR description